### PR TITLE
new test for submission and subject models,  base view for submission

### PR DESCRIPTION
**Pull request by BogVin**

### CODE REVIEW

File: app/submissions/urls.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from rest_framework import routers
+from .views import SubmissionViewList
+
+router = routers.DefaultRouter()
+router.register("", SubmissionViewList, basename="submissoin-list")
```
Comment (vladhoi) [inline app/submissions/urls.py:5]: typo in basename

File: app/tests/submissions/test_models.py [hunk 2]
```diff
@@ -49,3 +49,13 @@ def test_create_submission_without_user(create_valid_assigment):
...
             created_by=None,
         )
+
+
+@pytest.mark.django_db
+def test_delete_submission(create_valid_submission):
+    """
+    Ensure we can delete a submission.
+    """
+    s = create_valid_submission
```
Comment (vladhoi) [inline app/tests/submissions/test_models.py:59]: Rename variable `b` to something understandable

File: app/submissions/views.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from rest_framework import viewsets
+
+from .models import Submission
+from .serializers import SubmissionSerializer
+
+
+class SubmissionViewList(viewsets.ModelViewSet):
```
Comment (vladhoi) [inline app/submissions/views.py:7]: Couldn't find any tests for views

**Review by vladhoi — CHANGES_REQUESTED** [review]

**Review by doroshroman — COMMENTED** [review]
@BogVin, You need to be more careful with naming. Don't forget about tests

**Pull request closed without merging**